modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorInput.ts

```diff
@@ -23,6 +23,7 @@ import {
   SimpleSpec,
   Tabstopping,
   AlloyComponent,
+  Disabling,
```

Comment:
There's a trailing comma that probably shouldn't be there as this is the last item in the import list.